app/index.html

```diff
@@ -1,291 +1,398 @@
             </div>
-            <span class="cat-name">HTML</span>
-          </li>
-          <li class="category-item">
-            <div class="category-img">
-              <img class="cat-img" src="./assets/pngs/icons/css.jpeg" alt="">
+            <span class="dropdown-icon"
+              ><img src="./assets/pngs/icons/dropBtn.png" alt=""
+            /></span>
+          </span>
+          <div class="divide"></div>
+          <span class="section-title">
+            <div class="title-text">
+              <p>Skills</p>
             </div>
-            <span class="cat-name">CSS</span>
-          </li>
-        </ul>
-        <span class="section-title">
-          <div class="title-text">
-            <p>Frameworks</p>
+            <span class="dropdown-icon"
+              ><img src="./assets/pngs/icons/dropBtn.png" alt=""
+            /></span>
+          </span>
+          <div class="divide"></div>
+        </div>
+      </section>
+      <!-- about section ends here -->
+      <!-- contact section starts here -->
+      <section class="contact-section">
+        <div class="contact-container">
+          <div class="contact-title">
+            <h2>Contact Me</h2>
           </div>
-          <span class="dropdown-icon"><img src="./assets/pngs/icons/dropBtn.png" alt=""></span>
-        </span>
-        <div class="divide"></div>
-        <span class="section-title">
-          <div class="title-text">
-            <p>Skills</p>
+          <div class="contact-body">
+            <p>
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding i'd
+              love to help with it.
+            </p>
           </div>
-          <span class="dropdown-icon"><img src="./assets/pngs/icons/dropBtn.png" alt=""></span>
-        </span>
-        <div class="divide"></div>
-      </div>
-    </section>
-    <!-- about section ends here -->
-  </div>
-</body>
-
-</html>
\ No newline at end of file
+          <div class="form-container">
+            <form action="#">
+              <div class="input-container">
+                <input
+                  type="text"
+                  class="input-field"
+                  placeholder="Full Name"
+                  minlength="30"
+                  required
+                />
+              </div>
+              <div class="input-container">
+                <input
+                  type="email"
+                  class="input-field"
+                  placeholder="Email"
+                  required
+                />
+              </div>
+              <div class="input-container">
+                <textarea
+                  type="text"
+                  class="text-field"
+                  placeholder="Write your message here..."
+                  minlength="500"
+                  required
+                ></textarea>
+              </div>
+            </form>
```

Comment:
- [ ] Kindly make sure to add the `Get in touch` button or `input` with the `type` attribute which is set to `submit` value which will be the one that triggers the `form` submission once it is clicked.

- [ ] Kindly make sure to implement the `Get in touch` button interactions `(Active, hover, disabled, etc..)` styling.


    - ### YOUR DESIGN
   ![image](https://github.com/Amaramhdi1112/portfolio-amar-muha/assets/87197412/02d8b026-9b54-43e2-b64a-78247bec394a)

    - ### FIGMA DESIGN
    ![Screenshot from 2023-04-06 12-42-37](https://github.com/Amaramhdi1112/portfolio-amar-muha/assets/87197412/4139fe78-4ed1-4393-932d-5caecacf258c)

    
    - ### BUTTON INTERACTION STYLING
    ![image](https://user-images.githubusercontent.com/87197412/235726320-d0f52fe9-8163-47b6-a69e-976b67b314b9.png)